### ISSUE TITLE
Boot directory only 10.6 MB left

### ISSUE THREAD

**Issue by DamirRT** [subgraph/subgraph-os-issues#155]
Hello,

i got a error message after apt-get update/upgrade ...
Boot directory only 10.6 MB left
and a configuration error for the new kernel image 4.5.7-grsec-amd
... after reboot i got a kernel panic ...

![virtualbox_subgraph_13_06_2016_21_52_16](https://cloud.githubusercontent.com/assets/12501446/16021082/1231d328-31b1-11e6-9f02-b9ecce167c0c.png)

any hints ?

Cheers

**Comment by dma** [comment]
Yeah, you can remove stuff (and should). Sorry to reply so late. I just encountered this myself.

**Comment by ghost** [comment]
@dma Why are the not removed upon upgrade or with auto-remove? Ubuntu and Debian do this
